### DISCUSSION TITLE
Linking Another Folder to An Existing Project in MacOS

### DISCUSSION THREAD

**Discussion by GrimBlazer** [phcode-dev/phoenix#1934] [Q&A]
My project contains HTML files that use style sheets located in another folder on my Mac. These are separate because the style sheets are common to the website, whereas the project is for a specific set of HTML pages. Currently, I have the style sheets in both folders so the HTML will render.

Is there a way to LINK the CSS folder to my existing project? Thanks so much.

**Comment by devvaannsh** [comment]
Hey @GrimBlazer, to make everything work smoothly in Phoenix Code, your HTML and CSS files need to be in the same project folder. If the styles are shared, try opening the parent folder with all the files instead of duplicating them. That should fix the issue!
